src/fland1.c

```diff
@@ -1,24 +1,26 @@
 #include "sacramento.h"
+#include <R.h>
+#include <R_ext/Print.h>
 #include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
 
 #define EPS 1.e-5 /* Minimum parameter value */
 
 void fland1(struct SMA *sma, struct FSUM1 *fsum1) {
-
+  
```

Comment:
the linter is complaining about this insertion of whitespace.

Reply:
It is still complaining despite its removal

Reply:
All good now :)